pkgs/shells/zsh/zhooks/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchgit }:
+let
+  pname = "zhooks";
+  install_path = "share/zsh/${pname}";
+in
+stdenv.mkDerivation {
+  inherit pname;
+  version = "e6616b4a2786b45a56a2f591b79439836e678d22";
```

Comment:
Please see the readme file in pkgs how to properly name version packages fetched from commits